packages/redux/src/orders/actions/factories/fetchUserOrdersFactory.ts

```diff
@@ -19,18 +26,30 @@ import type { FetchOrdersAction } from '../../types/actions.types';
  */
 const fetchUserOrdersFactory =
   (getUserOrders: GetUserOrders) =>
-  (userId: number, query?: GetUserOrdersQuery, config?: Config) =>
-  async (dispatch: Dispatch<FetchOrdersAction>): Promise<Orders> => {
+  (userId: User['id'], query?: GetUserOrdersQuery, config?: Config) =>
+  async (dispatch: Dispatch<FetchOrdersAction>): Promise<OrderSummaries> => {
     try {
       dispatch({
         type: actionTypes.FETCH_USER_ORDERS_REQUEST,
       });
 
       const result = await getUserOrders(userId, query, config);
-      const normalizedResult = normalizeFetchOrdersResponse(result);
+      const normalizedResult = normalize<
+        OrderSummaryEntity,
+        {
+          orderSummaries: Record<
+            OrderSummaryEntity['merchantOrderCode'],
+            OrderSummaryEntity
+          >;
+        },
+        OrderSummariesNormalized
+      >(result, { entries: [orderSummary] });
+
+      const hashedQuery = hashFetchUserOrdersQuery(query);
 
       dispatch({
         payload: normalizedResult,
+        meta: hashedQuery,
```

Comment:
We usually use an object for the `meta` property like this:

```suggestion
        meta: { hash: hashedQuery },
```

So to be consistent with the other factories that use this, change it to be like that as well, please.

Reply:
Fixed 👍